frontend/public/locales/en/translation.json

```
@@ -1221,7 +1221,7 @@
   "discoveredPolicies.tooltip.clusterViolation": "Gatekeeper violations refer to the total number of pre-existing misconfigurations that do not comply with the defined constraints.",
   "discoveredPolicies.tooltip.gatekeeperRelatedResources": "Showing {{limit}} resource(s). To view more, increase the Gatekeeper constraint violations limit.",
   "discoveredPolicies.tooltip.responseAction": "In Open Cluster Management policies, response action refers to the remediationAction. For Gatekeeper, it refers to the enforcementAction. For ValidatingAdmissionPolicyBinding, it refers to the validationActions.",
-  "discoveredPolicies.tooltip.severity": "For Gatekeeper constraints, this value comes from the policy.open-cluster-management.io/severity annotation. For Kyverno policies, this value comes from the policies.kyverno.io/severity annotation.",
+  "discoveredPolicies.tooltip.severity": "In Open Cluster Management policies, the severity is specified in the `spec.severity` field. For Gatekeeper constraints, this value comes from the policy.open-cluster-management.io/severity annotation. For Kyverno policies, this value comes from the policies.kyverno.io/severity annotation.",
```

Probably omit the backticks for consistency since they aren't present elsewhere.

```suggestion
  "discoveredPolicies.tooltip.severity": "In Open Cluster Management policies, the severity is specified in the spec.severity field. For Gatekeeper constraints, this value comes from the policy.open-cluster-management.io/severity annotation. For Kyverno policies, this value comes from the policies.kyverno.io/severity annotation.",
```

---

But otherwise, I think it looks great! 🙂